nutribest.com/src/pages/equipments/[id].jsx

```diff
@@ -1,200 +0,0 @@
-import React, { useState} from 'react';
-import style from "./avijuice.module.css"
-import { useDispatch } from "react-redux";
-import { useRouter } from "next/router";
-import { addProduct } from '@/redux/SingleProduct/action';
-import { Flex, Box, Heading, Button, Text, useToast  } from '@chakra-ui/react';
-
-import Footer2 from 'Components/Footer/Footer2';
-
-
-const workflow1 = [
-  {
-    title: "Product Details",
-    content:
-      "MuscleBlaze Super Gainer XXL Chocolate is crafted especially for hardcore fitness enthusiasts who work hard to gain big and sturdy muscles and a sculpted physique. This advanced formula offers 15g protein per 100g serving. It has a blend of fast and slow-releasing proteins It offers 75g of simple and complex carbs per serving of 100g that ensures a robust and sustained calorie release in your body to fuel your muscles for a longer duration MuscleBlaze Super Gainer XXL is enriched with 27 vital vitamins and minerals that help in boosting the immunity and regulating enzymes in your body. Available in scrumptious chocolate flavor, MuscleBlaze Super Gainer XXL is free from any added sugars and trans fats",
-  },
-
-  {
-    title: "Product Information",
-    logo: "https://hkprod.s3.amazonaws.com/21232/bnr_2123154_o.jpg",
-    content:
-      "Get the insights you need to make decisions with confidence. Keep track of progress, timelines, and budgets with custom dashboards.",
-  },
-
-  {
-    title: "Other Information",
-    logo: "http://hkprod.s3.amazonaws.com/12075/normal_1207449_o.jpg",
-    content:
-      "MuscleBlaze Super Gainer XXL is a high-calorie sports nutrition supplement which is the perfect way to gain sturdy muscles. The unique blend of protein and complex carbs in 1:5 ratio provides with the fuel to muscles for quick recovery and muscle synthesis. The calories are derived from the clean and high-quality carbs and there is zero added sugar in it so that you can be sure that each serving is composed of nothing but the best. With the power of 27 vitamins and minerals, this mass gain formula is crafted to lift your muscles inches and weight by impressive numbers.",
-  },
-
-  {
-    title: "FAQ's",
-    content:
-      "Streamline processes to focus on the work that matters. Choose from a variety of automation recipes or create your own in minutes.",
-  },
-];
-
-
-
-
-const Page = ({product}) => {
```

Comment:
This component is same as Gainer and Juice page. Please make a common component and reuse it everywhere